training/checkpoint.py

```diff
@@ -49,8 +94,12 @@ def warm_start_model(checkpoint_path, model, ignore_layers=["embedding.weight"])
     model : Tacotron2
         tacotron2 model to load checkpoint into
     ignore_layers : list (optional)
-        list of layers to ignore (default is ['embedding.weight'])
-
+        list of layers to ignore (default is ["embedding.weight"])
+    symbols : list
+        list of text symbols used by the fresh model currently loaded
+    old_symbols : list (optional)
```

Comment:
I don't think we should pass this as a parameter. Can we not get it from the existing checkpoint each time?

Reply:
I can make that modification for new checkpoints, but not nvidia's as they do not save that information in the model file

Reply:
Let's make the assumption that we can either use the NVIDIA default or get it from the checkpoint for now if that's OK